src/gam.py

```diff
@@ -7742,6 +7745,13 @@ def _getAdminUserFromOAuth():
     credentials = storage.get()
   return credentials.id_token.get(u'email', u'Unknown')
 
+def _getDomainFromOAuth():
```

Comment:
Let's just take _getAdminUserFromOAuth() and generalize it so something like _getValueFromOAuth. You would specify 'email' or 'hd' (or anything else in credentials) and get that value returned.